modules/consent/lib/Consent/Store/Database.php

```diff
@@ -471,8 +484,13 @@ private function _getDB()
         if (isset($this->_timeout)) {
             $driver_options[PDO::ATTR_TIMEOUT] = $this->_timeout;
         }
+        if (isset($this->_options)) {
+            array_merge($driver_options, $this->_options);
```

Comment:
Gotcha, I suppose it is working for us without merging in $driver_options. Good catch.